### PR TITLE
Shugen performance fixes

### LINKED ISSUE #4: alpha channel black?
is there any alpha channel support?

### PR DESCRIPTION
**Pull request by campbellwmorgan**
Implements / improves changes in #6 by [@shugen002](https://github.com/shugen002)

* supersedes #6 
* closes #4 
* adds Chinese translation
* improves performance when no spout sender found

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -12,5 +16,20 @@ add_library(win-spout MODULE
...
+	add_custom_command(
+		TARGET win-spout POST_BUILD
+		COMMAND ${CMAKE_COMMAND} -E copy
+			"${CMAKE_SOURCE_DIR}/deps/spout/Binaries/${_src_suffix}/${targetfile}"
+			"${OBS_OUTPUT_DIR}/$<CONFIGURATION>/${OBS_PLUGIN_DESTINATION}/${targetfile}")
+endfunction()
 
 install_obs_plugin_with_data(win-spout data)
+copy_spout_file("SpoutLibrary.dll")
+copy_spout_file("SpoutLibrary.lib")
```
Comment (shugen002) [inline CMakeLists.txt]: Is it necessary to copu SpoutLibrary.lib to plugin folder?

**Comment by shugen002** [comment]
https://github.com/shugen002/obs-spout2-source-plugin/commit/5c575b534e130b3929b09ad4a2f6f779893527f8
I try OBS_EFFECT_DEFAULT is also acceptable but different with OBS_EFFECT_PREMULTIPLIED_ALPHA , so i add it.

File: CMakeLists.txt (hunk 1 above)
Comment (campbellwmorgan) [reply CMakeLists.txt]: I think you're right that it's unnecessary. I think it's only used in build right?

**Comment by shugen002** [comment]
wait #8

File: CMakeLists.txt (hunk 1 above)
Comment (shugen002) [reply CMakeLists.txt]: yes

**Comment by shugen002** [comment]
wait for a release and install package

**Comment by campbellwmorgan** [comment]
@shugen002 https://github.com/Off-World-Live/obs-spout2-source-plugin/releases/tag/v0.3

**Comment by shugen002** [comment]
I bring this plugin to my friends , they use OBS 24.0.3 and this cause OBS cannot start.
Did you build the release with the latest commited OBS code instead of latest released code ?

**Comment by campbellwmorgan** [comment]
Yes I built it against 25.x I'll do another build against 24.x today and release them both

**Comment by campbellwmorgan** [comment]
hey @shugen002 - would you mind asking your friends to try https://github.com/Off-World-Live/obs-spout2-source-plugin/releases/download/v0.3/OBS_Spout2_Plugin_Install_v0.3_mainline.24.0.6.exe and see if it works? Thanks

**Comment by shugen002** [comment]
it works.
But i want to point out that the latest Windows release version is 24.0.3 . And 24.0.6 is for mac .

**Pull request merged**